Submission for code review.

vzdrizhni opened the pull request:
Please review the Newsweek clone page made using bootstrap.

vzdrizhni committed:
Html and css added a lot

vzdrizhni committed:
fixed fonts

vzdrizhni committed:
Added, fixed and styled input section

vzdrizhni committed:
fixed nav responsivness

vzdrizhni committed:
first section added

vzdrizhni committed:
added third column

vzdrizhni committed:
Added middle section

vzdrizhni committed:
Finished main part of the page and applied some minor fixes

vzdrizhni committed:
Added first inside in-magazine section

vzdrizhni committed:
Added editor's pick section

vzdrizhni committed:
Added slideshow section

vzdrizhni committed:
Added more news section

vzdrizhni committed:
Added subscription section

vzdrizhni committed:
fixed subscribe section

vzdrizhni committed:
started footer

vzdrizhni committed:
Added footer

vzdrizhni committed:
Fixed subscription section

vzdrizhni committed:
Finished for large screens

vzdrizhni committed:
Some fixes

vzdrizhni committed:
Finished medium screens sizes

vzdrizhni committed:
Finished for a small screens

vzdrizhni committed:
Finished

vzdrizhni committed:
Added readme file

vzdrizhni committed:
Fixed project issue in readme file

vzdrizhni committed:
Fixed html validation

vzdrizhni committed:
updated title

vzdrizhni committed:
Updated livelink

pelzolga123 commented on style.css:
```diff
@@ -0,0 +1,790 @@
+}
+
+body {
+  background-color: #f4f5f6 !important;
```
Use of `!important` is considered a bad practice, please remove it from your project.

pelzolga123 reviewed (changes requested):
Project requires changes.

pelzolga123 commented:
## Hi Roman! ![](https://user-images.githubusercontent.com/38801423/62908122-703d4480-bd44-11e9-9115-42025ab7f341.png) 
### Your project looks :+1: amazing!
Still there are changes, you need to make, before this project can be approved:
* [x] Large version:
Bottom part is not aligned like on original website:
![image](https://user-images.githubusercontent.com/20416431/74555954-7ddabd80-4f65-11ea-9630-274c0a16f34e.png)
Original:
![image](https://user-images.githubusercontent.com/20416431/74556004-9ea31300-4f65-11ea-8b88-2cd991ebb90f.png)
* [ ] Tablet version:
* `Editor's pick` section doesn't look like original one:
![image](https://user-images.githubusercontent.com/20416431/74556174-20933c00-4f66-11ea-8ad9-bf4bc06d7619.png)
Original:
![image](https://user-images.githubusercontent.com/20416431/74556217-37d22980-4f66-11ea-8d1e-1deb0e3505c7.png)
* Next sections should be centered properly, image inside `newsletter` part takes near 25% of div, when text takes the rest of it  :
![image](https://user-images.githubusercontent.com/20416431/74556423-b29b4480-4f66-11ea-958a-f3671c162c69.png)
Original:
![image](https://user-images.githubusercontent.com/20416431/74556462-c777d800-4f66-11ea-97a9-8cb6db0808ce.png)
* Footer should look like original:
![image](https://user-images.githubusercontent.com/20416431/74556769-6f8da100-4f67-11ea-960f-371d06b4f1f9.png)
Original:
![image](https://user-images.githubusercontent.com/20416431/74556753-669ccf80-4f67-11ea-8b88-47ccc7db5ee3.png)

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*
* [ ] Mobile version:
* Sections logo looks too small:
![image](https://user-images.githubusercontent.com/20416431/74557104-22f69580-4f68-11ea-99d1-8aa2a6f36abd.png)
Original:
![image](https://user-images.githubusercontent.com/20416431/74557119-2d189400-4f68-11ea-9bff-e852a4f6d562.png)
* Next sections should be centered properly :
![image](https://user-images.githubusercontent.com/20416431/74557179-4f121680-4f68-11ea-859b-5294442b981f.png)
Original:
![image](https://user-images.githubusercontent.com/20416431/74557278-7963d400-4f68-11ea-8ef1-94ad4a3da5d6.png)
* Footer should look like original:
![image](https://user-images.githubusercontent.com/20416431/74557321-94cedf00-4f68-11ea-880d-6894e37f205f.png)
Original:
![image](https://user-images.githubusercontent.com/20416431/74557344-9d271a00-4f68-11ea-864a-ab6a6390e5fe.png)

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

vzdrizhni committed:
Fixed subscription section for large screens

vzdrizhni committed:
Merge branch 'feature' of https://github.com/vzdrizhni/Using-Bootstrap into feature

vzdrizhni committed:
Removed !important properties and adjusted subsccription section

vzdrizhni committed:
Fixed footer

vzdrizhni committed:
Fixed icon sizes and subscription section for smaller screens

vzdrizhni committed:
Updated livelink.

vzdrizhni commented:
Thanks for the review. Fixed issues you showed me.
- Fixed footer section for large, medium and small screens.
- Fixed Footer.
- Removed !important properties.
- Fixed icons size.
Thanks to @pelzolga123 
Please review it again.

mosesogwo reviewed (approved):
**Hi @vzdrizhni, you have done well implementing the requested changes:+1:.**
**Your project is approved and you can now merge to master:heavy_check_mark:**

Here are some recommendations to make your page better: 
 - On large and small screens, the images in the **Opinion** section have different sizes. You should make them of the same size.
- Remove the vertical scroll from the **Top Story** section
- Add some margin between the footer section and the section above it.

**Cheers:stars:**

### Status: Approved!

vzdrizhni commented:
Awesome! Thank you!
